### ISSUE TITLE
Unable to add Mastodon.social profiles

### ISSUE THREAD

**Issue by yannickdoteu** [1Feed/feedback#83]
Hey! I can't seem to add profiles from mastodon.social. other instances like volt.cafe or tukkers.online seem to be fine though. 

(But 90% of profiles is on mastodon.social anyway so I need it.l)

Thanks a lot!

**Comment by Booligoosh** [comment]
Hi @yannickdoteu, this is now fixed, really sorry for the delay!!

I believe this was due to mastodon.social moving to an early release candidate of Mastodon version 4 at the time. There is a bug in Mastodon v4 (https://github.com/mastodon/mastodon/issues/20770) that will only be fixed in the next version, where it does not include the correct metadata for RSS readers to find the RSS feed URL.

As the new version isn't out yet and some instances may stay on a buggy version for longer, I have added a workaround to 1Feed where it'll manually construct the feed URL if it doesn't see it there as it should be.
